include/Core/Base/Mixin/Renderer.php

```diff
@@ -85,10 +85,42 @@ protected function split_post_type_and_status( $str ) {
 		return $type_status;
 	}
 
+	/**
+	 * Render User role dropdown.
+	 */
+	protected function render_user_role_dropdown() {
+		$users_count = count_users();
+?>
+    <tr>
+        <td>
+        	<select class="select2" name="smbd_<?php echo esc_attr( $slug ); ?>_post_type">
```

Comment:
You should use `$this->slug` instead of just `$slug`.

Replace it in this function and also in `render_post_type_dropdown` function